### ISSUE TITLE
Missing Method

### ISSUE THREAD

**Issue by itzmxritz** [modnext/mouseSteering#19]
### Version

1.1.1.0

### Game Version

1.14.0.0

### What happened?

On FS24, the game is stuck at 50% loading, appearing following error in log output

### Relevant log output

```shell
:/Users/Moritz/Documents/My Games/FarmingSimulator2025/mods/FS25_mouseSteering/src/gui/MouseSteeringMenu.lua:35: attempt to call missing method 'registerControls' of table
```

**Comment by aaw3k** [comment]
The modification has not been adapted by me for Farming Simulator 2025 in any way. I am waiting for Giants to release the gameSource so I can prepare the game for the new version. Any materials available on external websites, other than ModHub and GitHub, are not created or modified by me.

**Comment by garik-59** [comment]
scriptBindingChanges
removed
        getFogPlaneHeight
        getFogPlaneMieScale
        setDropCountScale
        FrameElement:registerControls (all gui elements with an id in xml config are automatically added to the namespace)

**Comment by garik-59** [comment]
> Модифікація жодним чином не адаптована мною для Farming Simulator 2025. Я чекаю, поки Giants випустять gameSource, щоб я міг підготувати гру до нової версії. Будь-які матеріали, доступні на зовнішніх веб-сайтах, окрім ModHub і GitHub, не створюються та не змінюються мною.

you say about Farming Simulator 25\sdk\debugger  gameSource.zip ?

**Comment by aaw3k** [comment]
Yes, I know. I have already familiarized myself with the resources and I'm aware that GameSource is already available. I'm sure mouse steering will be introduced to FS25 soon.

**Comment by garik-59** [comment]
I'm glad to hear that, thanks, maybe there will be a desire and opportunity to add a new function of adjusting the RPM with the mouse wheel

**Comment by garik-59** [comment]
do you have progress?

**Comment by aaw3k** [comment]
I'm currently working on an entirely new GUI.

**Comment by garik-59** [comment]
> I'm currently working on an entirely new GUI.

very nice, I have a question, how did you come to create this mod, how did you figure out how to use the game API?
if it's not difficult, give an answer

**Comment by aaw3k** [comment]
The mod was created out of a need for smoother control without using a steering wheel. By analyzing the gameSource files, I was able to develop the modification. If you know a programming language like Lua and have basic logical thinking skills, you can create your own modifications as well.

**Comment by garik-59** [comment]
> Мод був створений через необхідність більш плавного керування без використання керма. Проаналізувавши файли gameSource, я зміг розробити модифікацію. Якщо ви знаєте таку мову програмування, як Lua, і маєте базові навички логічного мислення, ви також можете створювати власні модифікації.

ok, thanks

**Comment by garik-59** [comment]
> The mod was created out of a need for smoother control without using a steering wheel. By analyzing the gameSource files, I was able to develop the modification. If you know a programming language like Lua and have basic logical thinking skills, you can create your own modifications as well.

hi, how is the work on the mod going

**Comment by SpaceGame123** [comment]
nice to know you are updating 😄

**Comment by Cyber-Syntax** [comment]
It's great to hear that you're updating! If you make it a public repository for FS25, we might be able to assist you. Unfortunately, I'm not familiar with Lua, but I could possibly help with translation or other tasks.

**Comment by DaStrobel** [comment]
Thank you for creating this script! Looking forward to us eit in FS25 as well!

**Comment by SpaceGame123** [comment]
how's it going @aaw3k  ?

**Comment by aaw3k** [comment]
I just sent FS25 mod to ModHub — it’ll be out shortly.
